test/unit/sequence_base_test.rb

```diff
@@ -333,15 +333,6 @@ class SequenceBaseTest < MiniTest::Test
       assert all_resources.map(&:id) == ['1', '2']
     end
 
-    it 'fails on 404' do
```

Comment:
This means we no longer fail a server if the next link in a Bundle is broken.

Reply:
Yes. the next_bundle returns nil if the next_link is empty or the server returns 4xx

Reply:
I think that is a problem.

Reply:
Restored the unit test and changed the fetch_all_bundled_resources() to handle that